manifester/manifester.py

```diff
@@ -12,17 +12,23 @@
 
 class Manifester:
     def __init__(self, manifest_category, allocation_name=None, **kwargs):
-        self.allocation_name = allocation_name or "".join(random.sample(string.ascii_letters, 10))
-        self.offline_token = kwargs.get("offline_token", settings.offline_token)
-        manifest_data = settings.manifest_category.get(manifest_category)
-        self.subscription_data = manifest_data.subscription_data
-        self.sat_version = kwargs.get("sat_version", manifest_data.sat_version)
+        self.allocation_name = allocation_name or "".join(
+            random.sample(string.ascii_letters, 10)
+        )
+        self.manifest_data = settings.manifest_category.get(manifest_category)
+        self.offline_token = kwargs.get(
+            "offline_token", self.manifest_data.offline_token
+        )
```

Comment:
I think it would be a good idea to modify this so the offline token could be set at the top level or overridden at the manifest level. That way someone using the same token for each doesn't need to duplicate the setting for each manifest.
```suggestion
        self.offline_token = kwargs.get(
            "offline_token", self.manifest_data.get("offline_token", settings.offline_token)
        )
```